### ISSUE TITLE
PR image nicht angezeigt

### ISSUE THREAD

**Issue by OKienzler** [saw-leipzig/kucha#436]
Images in PR werden nicht angezeigt:
![grafik](https://github.com/user-attachments/assets/87e60ea3-cbca-456e-bcad-a1cfbf56db27)

**Comment by erikradisch** [comment]
war das eventuell was vorübergehendes? Zumindest kann ich den Fehler nicht reproduzieren. Bei mir geht's

**Comment by OKienzler** [comment]
Ja, war was vorübergehendes, jetzt sind die, die ich angelegt habe zu sehen. Aber z.B. dieses immer noch nicht, war erst neu von Dominika am 14.12.2024 hochgeladen bzw. geändert. Image ID. 30571

![grafik](https://github.com/user-attachments/assets/0c38e5bb-4ba1-4f2c-bfe7-bd6712cf549c)
